LEGO1/mxmidimanager.h

```diff
@@ -0,0 +1,42 @@
+#ifndef MXMIDIMANAGER_H
+#define MXMIDIMANAGER_H
+
+#include "decomp.h"
+#include "mxaudiomanager.h"
+
+// VTABLE 0x100dc930
+// SIZE 0x58
+class MxMIDIManager : public MxAudioManager
+{
+public:
+  MxMIDIManager();
+  virtual ~MxMIDIManager() override;
+
+  void DeinitializeMIDI();
+  MxS32 CalculateVolume(MxS32 p_volume);
+  void SetMIDIVolume();
+
+  virtual void Destroy();                                                   // vtable+18

```

Comment:
Could you add the `override` keyword to the virtual functions that are overridden? Thanks!